rest-api/src/test/scala/za/co/absa/enceladus/rest_api/integration/controllers/DatasetApiIntegrationSuite.scala

```diff
@@ -437,4 +436,48 @@ class DatasetApiIntegrationSuite extends BaseRestApiTestV2 with BeforeAndAfterAl
     }
   }
 
+  s"DELETE $apiUrl/disable/{name}/{version}" can {
```

Comment:
Note, that this is the original V2 API - adding test cases here as a regression guard